docs/journalist/submissions.rst

```diff
@@ -56,26 +56,33 @@ administrator. The printer must be plugged into the computer's USB port.
 Exporting to an Export USB
 --------------------------
 
-Currently, a LUKS-encrypted USB drive is required for exporting submissions. A
-Linux-based system such as Tails is required to configure and use a LUKS-encrypted
-drive, meaning that for the time being, you will only be able to
-export to a Linux environment where these drives can be read. For assistance
-with this, see your SecureDrop administrator.
-
-Once you have provisioned a LUKS-encrypted export drive, insert the drive and
-click **Export**.
-
-|screenshot_export_dialog|
-
-You will be prompted for the password configured for this
-USB drive.
-
-|screenshot_export_drive_passphrase|
-
-Once you see a message informing you that the export was successfully completed,
-you can safely unplug the USB drive. Alternatively, you can leave the drive
-plugged in and export additional files.
-
+Currently, a LUKS- or VeraCrypt-encrypted USB drive is required for exporting submissions.
+
+1. Insert the USB drive and wait for the ``sd-devices`` VM to start.
+2. If your drive is using VeraCrypt, you will need to unlock it manually:
+
+   1. Open the file menu by clicking on the **Q** application menu (in the top left),
+      select **sd-devices** and click **Files**.
+   2. In the left sidebar, there should be an entry labeled **# GB Possibly Encrypted**,
+      click it.
+      |screenshot_veracrypt_sd_devices_files|
+   3. You will be prompted for the password configured for this USB drive:
+      - Volume type: leave both unchecked
+      - PIM: leave empty
+      - Password: drive's password
+      - Forget password immediately: selected
```

Comment:
this is a nit, but: rst jumbles up this formatting a bit.  Also, the PIM and Hidden Volume, it depends how they set it up. If they changed the PIM when configuring the drive, they need to put in their custom number here.  And in theory if they unlock the hidden volume here we can probably export to it. So I wasn't sure whether to put our suggestions or just put "Settings consistent with original configuration" or something

![Screenshot 2024-03-19 at 13-30-08 Working with submissions — SecureDrop Workstation 0 0 1 documentation](https://github.com/freedomofpress/securedrop-workstation-docs/assets/22777700/21058165-d005-437c-8a0f-e198f95019ae)

Reply:
> this is a nit, but: rst jumbles up this formatting a bit.

oops, I was so focused on the images I missed this. fixed.

>  Also, the PIM and Hidden Volume, it depends how they set it up. If they changed the PIM when configuring the drive, they need to put in their custom number here

I was thinking that this should match the drive creation steps (which now I realize should be more specific about leaving PIM, etc. blank). We could also have it be like `PIM: leave empty (unless you set one during creation)`?

Reply:
Alternatively:
```patch
diff --git a/docs/journalist/submissions.rst b/docs/journalist/submissions.rst
index ec93355..b9ca79e 100644
--- a/docs/journalist/submissions.rst
+++ b/docs/journalist/submissions.rst
@@ -66,15 +66,11 @@ Currently, a LUKS- or VeraCrypt-encrypted USB drive is required for exporting su
    2. In the left sidebar, there should be an entry labeled **# GB Possibly Encrypted**,
       click it.
       |screenshot_veracrypt_sd_devices_files|
-   3. You will be prompted for the password configured for this USB drive:
-
-      - Volume type: leave both unchecked
-      - PIM: leave empty
-      - Password: drive's password
-      - Forget password immediately: selected
-
+   3. You will be prompted for the password configured for this USB drive. In most cases
+      you will just need to enter the drive's password and leave all the other fields blank
+      unless you picked different settings during provisioning. Make sure you select
+      **Forget password immediately** and then click **Connect**.
       |screenshot_veracrypt_sd_devices_files_unlock|
-   4. Click **Connect**.
 
 3. Back in your source's conversation, click **Export**.
    |screenshot_export_dialog|
```

Reply:
And maybe have the `select **Forget password immediately** and then click **Connect**` be step 4 so it doesn't get overlooked.

Reply:
(per call, resolving - lgtm and I agree it's simpler to leave as is in case one person provisions, one person uses the drive)